File: packages/python3/model/ball.py
class Ball:
    def __init__(self, json):
        self.x = json.get("x")
        self.y = json.get("y")
        self.z = json.get("z")
        self.velocity_x = json.get("velocity_x")
        self.velocity_y = json.get("velocity_y")
        self.velocity_z = json.get("velocity_z")
        self.radius = json.get("radius")

File: packages/javascript/model/Ball.js
/**
 * Created by Quake on 18.12.2018
 */
'use strict';

function Ball() {

}

Ball.prototype.read = function(json) {
    if (typeof json == 'string') {
        json = JSON.parse(json);
    }

    this.x = json.x;
    this.y = json.y;
    this.z = json.z;
    this.velocity_x = json.velocity_x;
    this.velocity_y = json.velocity_y;
    this.velocity_z = json.velocity_z;
    this.radius = json.radius;
}

module.exports = Ball;